src/Stan.hs

```diff
@@ -55,14 +55,17 @@ runStan StanArgs{..} = do
     let useDefConfig = maybe True snd (trialToMaybe defConfTrial)
     -- config
     tomlConfig <- getTomlConfig useDefConfig stanArgsConfigFile
-    let config = finaliseConfig $ defaultConfig <> tomlConfig <> stanArgsConfig
-    putTextLn $ prettyPrintTrial config
+    let configTrial = finaliseConfig $ defaultConfig <> tomlConfig <> stanArgsConfig
+    putTextLn $ prettyPrintTrial configTrial
 
     hieFiles <- readHieFiles stanArgsHiedir
     -- create cabal default extensions map
     cabalExtensionsMap <- createCabalExtensionsMap stanArgsCabalFilePath hieFiles
+    -- get checks for each file
+    let config = fromMaybe (error "asd") $ trialToMaybe configTrial
```

Comment:
Is it possible, that the final `configTrial` can be `Fiasco`? We have `defaultConfig` which is `pure []`. I know, that we can't prove the compiler that we won't have `Fiasco` at the end, I'm just curious, that maybe I'm missing something.

Reply:
It is not possible from my point of view. What do you you propose to do then? Do not have `Trial` here?

Reply:
No, `Trial` is very needed here. I'm just thinking about changing error message to `error "This is Fiasco, bro"`